Tella/Data/VaultManager/VaultFilesManagerExtension.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  VaultFilesManagerExtension.swift
+//  Tella
+//
+//  Created by Dhekra Rouatbi on 13/6/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+import MobileCoreServices
+import Photos
+
+extension VaultFilesManager {
+    
+    func updateURL(importedFile:inout ImportedFile) async {
+        
+        do {
+            
+            var url : URL?
+            guard let asset = importedFile.asset else {
+                return
+            }
+            
+            let mediaType = asset.mediaType
```

Comment:
We can remove this and call `asset.mediaType` directly  in the if condition below.